ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVTest.scala

```diff
@@ -236,7 +234,7 @@ object KVTest {
   def submitConfig(
       configModify: Configuration => Configuration,
       submissionId: SubmissionId = randomLedgerString,
-      mrtDelta: Duration = minMRTDelta
+      mrtDelta: Duration = minMRTDelta,
```

Comment:
we can probably remove this as well and furthermore remove `maxRecordTime` from https://github.com/digital-asset/daml/blob/f67c32b002fd28a68e6cfd279ec1782db4a07336/ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueSubmission.scala#L121-L122